tbx/core/blocks.py

```diff
@@ -72,15 +68,6 @@ class Meta:
         max_num = 1
 
 
-class KeyPoint(StructBlock):
-    title = CharBlock()
-    intro = CharBlock()
-    link = PageChooserBlock()
-
-    class Meta:
-        icon = "form"
-
-
 class ImageFormatChoiceBlock(FieldBlock):
```

Comment:
This can also be removed, with the corresponding `alignment` field removed on `ImageBlock`

Reply:
Ah, just trying this now and we'll get the same issue as before
<img width="759" alt="image" src="https://github.com/torchbox/torchbox.com/assets/68611528/69c9f719-7208-43cc-9ef5-aaed35f13cca">
Also, is Helen right about delaying its deletion until after we've imported the old database? https://torchbox.monday.com/boards/1192293412/pulses/1361473029?userId=34000784&blockId=5a836508-45f1-4455-a4f5-8d730257d9e6

Reply:
I see, then I suppose we can leave it for now